ms-ml/app/main.py

```
@@ -0,0 +1,55 @@
+import time
+
+logging.basicConfig(level=settings.log_level)
+logger = logging.getLogger(__name__)
+
+@asynccontextmanager
+async def lifespan(app: FastAPI):
+    # Startup
+    logger.info("Starting ZiBooka ML Service...")
+    try:
+        await mongodb.connect()
+    except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)